internal/component/postgresql/postgresdbaccess.go

```diff
@@ -335,7 +330,7 @@ func (p *PostgresDBAccess) doDelete(parentCtx context.Context, db dbquerier, req
 			return state.NewETagError(state.ETagInvalid, err)
 		}
 
-		result, err = db.Exec(parentCtx, "DELETE FROM state WHERE key = $1 AND xmin = $2", req.Key, uint32(etag64))
+		result, err = db.Exec(parentCtx, fmt.Sprintf("DELETE FROM state WHERE key = $1 AND %s = $2", p.etagColumn), req.Key, uint32(etag64))
```

Comment:
In hot paths we prefer to use string concatenation rather than fmt.Sprintf for simple concatenation operations

Reply:
Thanks, makes sense